NIFI-9367 Update README.md (#5511)
apache/nifi: nifi-docker/dockerhub/README.md

This will provide a running instance, exposing the instance UI to the host system on at port 8443,
viewable at `https://localhost:8443/nifi`.

> It will generate credentials, which could be found in the docker logs after the startup phase (about a minute):

Reviewer: Recommend adjusting the wording as follows, similar to the main project README:

```suggestion
The default configuration generates a random username and password on startup. NiFi writes the generated credentials to the application log.

The following command can be used to find the generated credentials on operating systems with grep installed:
```

    docker logs nifi | grep Generated

> the credentials will then be displayed such as

Reviewer: Following the project README:
```suggestion
NiFi logs the generated credentials as follows:
```

>     Generated Username [5ae5ed8e-b695-4589-bfb0-aee8989a08b3]
>     Generated Password [YJZD3kP8SFzF+SbRmHlcTBifiZTvS6bb]

Reviewer: ```suggestion
    Generated Username [USERNAME]
    Generated Password [PASSWORD]
```

      -e NIFI_WEB_HTTPS_PORT='9443' \
      apache/nifi:latest

> The esiest way to run NiFi is probably with specifying Single User Authentication credentials using environment variables as follows:

Reviewer: Recommend reverting this change as the existing wording is already clear.

      -e SINGLE_USER_CREDENTIALS_PASSWORD=ctsBtRBKHRAx69EqUghvvgEvjnaLjFEB \
      apache/nifi:latest

> Please note that the password must be 12 chars minimmum, or a new one will be generated (see above to find it in the logs)

Reviewer: The wording should be adjusted to correct the spelling of `minimum`, and spelling out `characters` would be helpful.
```suggestion
Please note that the password must be 12 characters minimum, otherwise NiFi will generate a random username and password.
```
